src/components/inputs/Dates/Calendar/Calendar.test.tsx

```diff
@@ -45,7 +45,11 @@ describe('Calendar', () => {
     const minDate = new Date(2022, 8, 26);
 
     renderWithThemeProvider(
-      <Calendar data-testid="calendar" minDate={minDate} />
+      <Calendar
```

Comment:
I think you want these out of this PR

Reply:
Oh yeah, my intention was the merge the other one first lol